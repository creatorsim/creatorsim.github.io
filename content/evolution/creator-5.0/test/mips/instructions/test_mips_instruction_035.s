
# Creator (https://creatorsim.github.io/creator/)
#

.text
main:

    li $t0, 23457464
    li $t1, 222
    li $t2, 0xFABB53
    li $t3, -23457475
    
    addu $t4, $t0, $t1
    addu $t5, $t1, $t2
    addu $t6, $t2, $t3
    addu $t7, $t3, $t0