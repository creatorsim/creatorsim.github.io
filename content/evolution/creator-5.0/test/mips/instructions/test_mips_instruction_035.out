
cr[PC]:0x38; ir[1,at]:0xfe9a113d; ir[8,t0]:0x165eeb8; ir[9,t1]:0xde; ir[10,t2]:0xfabb53; ir[11,t3]:0xfe9a113d; ir[12,t4]:0x165ef96; ir[13,t5]:0xfabc31; ir[14,t6]:0xff94cc90; ir[15,t7]:0xfffffff5; keyboard[0x0]:''; display[0x0]:''; 

